docs/manual/kinds/vr-csr.md

```diff
@@ -35,12 +35,19 @@ Cisco CSR1000v node launched with containerlab can be managed via the following
 !!!info
     Default user credentials: `admin:admin`
 
+## Interface naming
+
+CSR1000v nodes use the interface naming convention `GigabitEthernetX` (or `GiX`, both are accepted), where X denotes the port number.
+
+!!!warning
+    Data port numbering starts at `2`, as `Gi1` is reserved for management connectivity. Attempting to use `Gi1` in a containerlab topology will result in an error.
+
```

Comment:
Hi @vista- 
regarding this note, I think @ipspace tied this to an issue with the way csr1000v was handling changing mac addresses

https://github.com/hellt/vrnetlab/pull/206

it seems with https://github.com/hellt/vrnetlab/pull/206 merged we should expect csr1000v to map `eth1` to `Gi1`

Reply:
CSR interfaces start with Gi1, which is the management interface (mapped into eth0), so it's true that the first data-plane interface is Gi2 (mapped into eth1). The problem I fixed in in hellt/vrnetlab#206 was CSR mapping the first interface into Gi2 (because of changed MAC address)

Reply:
ah, I see, thanks for clarifying @ipspace